frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -136,44 +128,48 @@ export const RequestedChangesDisplay = ({
         data &&
         data.length &&
         data.some((responseItem) => responseItem.message);
-
     return (
         <>
-            <StaticAlert type={alertType} heading={heading} message={message}>
-                <h5 className="margin-bottom-1">Resources</h5>
-                <ul className={"margin-0"}>
-                    <li>
-                        <USLink
-                            target="_blank"
-                            href="/resources/programmers-guide"
-                        >
-                            ReportStream Programmers Guide
-                        </USLink>
-                    </li>
-                    <li>
-                        <USLink href="https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html">
-                            LOINC In Vitro Diagnostic (LIVD) Test Code Mapping
-                        </USLink>
-                    </li>
-                </ul>
-            </StaticAlert>
+            <StaticAlert type={alertType} heading={heading} message={message} />
             {showTable && (
                 <>
-                    <h3>{title}</h3>
+                    <div className="display-flex flex-justify">
+                        <h3>{title}</h3>
+                        <Button
+                            className="usa-button flex-align-self-start height-5 margin-top-4 usa-button--outline"
+                            type={"button"}
+                            onClick={() => {
```

Comment:
Let's break out the CSV exporting logic into a generic function that can be used anywhere.  We also have custom code [in OrgsTable.tsx](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/components/Admin/OrgsTable.tsx#L56-L84) that could use this function in the future -- we can add a ticket for that once we have a generic function, though.